packages/content/src/requests/items/createAttribute/CreateAttributeAcceptResponseItem.ts

```diff
@@ -17,7 +17,7 @@ export class CreateAttributeAcceptResponseItem extends AcceptResponseItem implem
     @validate()
     public attributeId: CoreId;
 
-    public static override from(value: ICreateAttributeAcceptResponseItem | CreateAttributeAcceptResponseItemJSON): CreateAttributeAcceptResponseItem {
+    public static override from(value: ICreateAttributeAcceptResponseItem | Omit<CreateAttributeAcceptResponseItemJSON, "@type">): CreateAttributeAcceptResponseItem {
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here